examples/ssh/linux_updates.js

```diff
@@ -74,7 +74,9 @@ function executeCommand(command) {
 function validate() {
     console.info("Verifying device can respond correctly to command ... ");
     executeCommand(fullCommand)
-        .then(D.success)
+        .then(function(){
```

Comment:
This does not correctly verify if the device supports the needed commands
(eg any device we connect to via SSH will pass)
A better approach will be to try to parse either the full command result/output (for any errors) and/or execute a specific validate-centric command and parse it without any output in the resolve so that you can keep the .then(D.success) after it